measure_extinction/modeldata.py

```diff
@@ -209,7 +209,7 @@ def stellar_sed(self, params, velocity=None):
 
         return sed
 
-    def dust_extinguished_sed(self, params, sed, velocity=0.0):
+    def dust_extinguished_sed(self, params, sed, fit_range="all", velocity=0.0):
```

Comment:
The fit_range parameter needs to be added to the docstring info right after the def line.